platform/viewer/cypress/integration/common/OHIFCornerstoneToolbar.spec.js

```diff
@@ -472,4 +478,100 @@ describe('OHIF Cornerstone Toolbar', () => {
     cy.get('@viewport').click(x2, y2, { force: true });
     cy.get('.measurementItem').should('not.exist');
   });
+
+  it('check if Invert tool will change the colors of the image in the viewport', () => {
+    // Click on More button
+    cy.get('@moreBtn').click();
+    // Verify if overlay is displayed
+    cy.get('.tooltip-toolbar-overlay').should('be.visible');
+
+    // Click on Invert button
+    cy.get('.tooltip-inner')
+      .contains('Invert')
+      .click();
+
+    // check on cornerstone if image was inverted
+    cy.window()
+      .its('cornerstone')
+      .then($cornerstone => {
+        expect($cornerstone.getEnabledElements()[0].viewport.invert).to.be.true;
+      });
```

Comment:
If we're doing the visual test here, I think we can remove the cornerstone test. We're close to "turning on" visual tests for PR workflows.